src/raft.rs

```diff
@@ -147,7 +146,37 @@ pub struct Config {
     pub tag: String,
 }
 
+impl Default for Config {
+    fn default() -> Self {
+        const HEARTBEAT_TICK: usize = 150;
```

Comment:
Looks like we need to tick 150 times to send a heartbeat, maybe too long.

Reply:
TiKV uses 2, I'll update it to that. https://github.com/pingcap/tikv/blob/2b3f52d35adebd8d95fa052dd58b33338a44aae0/src/raftstore/store/config.rs#L138